.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+    - devel
+    paths_ignore:
+    - "LICENSE"
+    - "**.md"
+    - "**.rst"
+  pull_request:
+    branches:
+    - master
+    - devel
+    paths-ignore:
+    - "LICENSE"
+    - "**.md"
+    - "**.rst"
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        # consider using these
+        # os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt install openbabel
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install tox tox-gh-actions
```

Comment:
Py3.6 `.travis/install_cplex.sh`:
- Mock and check that secret is not showed in log.
- Adapt it to python3.8.

Reply:
CPLEX path and unpacking must be corrected.